src/lib.rs

```diff
@@ -1,9 +1,9 @@
 #![deny(
-    missing_docs,
```

Comment:
It does not, it's just a byproduct of me trying out a few things and putting the flag back in the wrong place. Line 28 is where the fix actually is.